main.js

```diff
@@ -130,13 +130,56 @@ const projectsArray = [
   },
 ];
 
+for (let i = 0; i < projectsArray.length; i += 1) {
+  const projectWrapper = document.getElementsByClassName('project_wrapper')[0];
+
+  const projectCard = document.createElement('article');
+  projectCard.classList.add('project_card', `card${i}`);
+
+  const projectCardImage = document.createElement('img');
+  projectCardImage.classList.add('project_image');
+  projectCardImage.setAttribute('src', projectsArray[i].imgMobile);
+  projectCardImage.setAttribute('alt', projectsArray[i].name);
+  projectCard.appendChild(projectCardImage);
+
+  const projectCardWrapper = document.createElement('div');
+  projectCardWrapper.classList.add('project_card_wrapper');
+  projectCard.appendChild(projectCardWrapper);
+
+  const projectCardTitle = document.createElement('h4');
+  projectCardTitle.classList.add('project_card_title');
+  projectCardTitle.innerHTML = projectsArray[i].name;
+  projectCardWrapper.appendChild(projectCardTitle);
+
+  const projectCardTags = document.createElement('ul');
+  projectCardTags.classList.add('project_card_tags');
+  projectCardWrapper.appendChild(projectCardTags);
+
+  for (let j = 0; j < projectsArray[i].techTags.length; j += 1) {
+    const projectCardTag = document.createElement('li');
+    projectCardTag.innerHTML = projectsArray[i].techTags[j];
+    projectCardTags.appendChild(projectCardTag);
+  }
+
+  const projectCardButton = document.createElement('button');
+  projectCardButton.classList.add('project_card_button');
+  projectCardButton.setAttribute('type', 'button');
+  projectCardButton.innerText = 'See Project';
+
+  projectCardWrapper.appendChild(projectCardButton);
+
+  projectWrapper.appendChild(projectCard);
+}
+
+// Pop-up Window //
+
 function createPopup(index) {
```

Comment:
Very well done the popup window shows when we click see project button 👍🏼  , I have only  two small styling issues to fix 

- [ ] reduce the padding of see live and see source buttons and make sure the spacing between title and these buttons is same as in the design reducing button padding and title size will create that space , and center the vertical alignment of title and buttons container.
- [ ] reduce the opacity of the popup container to give show the page in the background 


**your design**
![Capture d’écran du 2023-07-28 21-13-36](https://github.com/mateoburvi/Portfolio/assets/59707859/7071d628-f7aa-4e3b-b1eb-7b358dd687ac)

**Figma design**
![Capture d’écran du 2023-07-28 21-13-15](https://github.com/mateoburvi/Portfolio/assets/59707859/cc650859-9c94-4f33-b4b4-887019d90ad5)